pkg/api/validation/edgeconnect/api_server.go

```diff
@@ -11,6 +12,16 @@ const (
 	errorInvalidAPIServer = `The EdgeConnect's specification has an invalid apiServer value set.
 	Make sure you correctly specify the apiServer in your custom resource.
 	`
+
+	errorMissingAllowedSuffixAPIServer = `The EdgeConnect's specification has an invalid apiServer value set.
+	Missing required suffix.
```

Comment:
What do you think about  ?
```
The domain is invalid. Allowed domains are: a , b , c.
```

Reply:
It's not about allowed domains, it's more about the prefix.